settings.gradle.kts

```diff
@@ -1,34 +1,32 @@
-val nodeModulesMsdk = File("${rootProject.projectDir}/libs/SalesforceReact/node_modules/react-native/android") // For stand-alone MSDK builds.
-val nodeModulesTemplate = File("${rootProject.projectDir}/../../node_modules/react-native/android") // For template app builds.
-val hasNodeModules = nodeModulesMsdk.exists() || nodeModulesTemplate.exists()
+import org.gradle.api.initialization.resolve.RepositoriesMode.FAIL_ON_PROJECT_REPOS
+
+pluginManagement {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+dependencyResolutionManagement {
+    repositoriesMode.set(FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+rootProject.name = "SalesforceMobileSDK-Android"
 
 include("libs:SalesforceAnalytics")
 include("libs:SalesforceSDK")
 include("libs:SmartStore")
 include("libs:MobileSync")
 include("libs:SalesforceHybrid")
-if (hasNodeModules) {
-    include("libs:SalesforceReact")
-} else {
-    logger.warn("Skipping `SalesforceReact` module since local node repository is missing.  Has `yarn install` been run in the directory containing `package.json`?")
-}
+include("libs:SalesforceReact")
 include("hybrid:HybridSampleApps:AccountEditor")
 include("native:NativeSampleApps:AppConfigurator")
 include("native:NativeSampleApps:ConfiguredApp")
 include("native:NativeSampleApps:MobileSyncExplorer")
 include("hybrid:HybridSampleApps:MobileSyncExplorerHybrid")
 include("native:NativeSampleApps:RestExplorer")
-
-dependencyResolutionManagement {
```

Comment:
While I was removing the outdated React Native repository configuration, I also compared this with the default `settings.gradle.kts` of a new Android Studio project for consistency.  Based on that, I moved this to the top of the file to match that for easier comparison moving forward plus restored some other Android Studio defaults.